webserver/cWebem.cpp

```diff
@@ -28,6 +28,8 @@
 #define SHORT_SESSION_TIMEOUT 600 // 10 minutes
 #define LONG_SESSION_TIMEOUT (30 * 86400) // 30 days
 
+#define HTTP_DATE_RFC_1123 "%a, %d %b %Y %H:%M:%S %Z" // Sun, 06 Nov 1994 08:49:37 GMT
```

Comment:
Is this different then using the make_web_time function ? 
If it is, one of the two is correct ?

Reply:
Don't really know. Seems similar. The define is copied from request_handler.cpp and it seemed to explain the hardcoded entry that was in the original cWebem.cpp on line 2101 (which happened to be an override on a header field that already exists)

Reply:
Looks like the strftime_t function is actually the one that should be preferred as it uses the thread safe localtime_r() function where make_web_time() uses the old gmtime() function. Not sure which one actually returns the time we want and is not distorted by a TZ offset.